webpack.config.js

```diff
@@ -186,23 +186,27 @@ const config = {
     // Each output page needs a HTMLWebpackPlugin entry
     new HTMLWebpackPlugin({
       filename: "index.html",
-      template: path.join(__dirname, "src", "lobby.html"),
+      template: path.join(__dirname, "src", "index.html"),
       // Chunks correspond with the entries you wish to include in your html template
-      chunks: ["lobby"]
+      chunks: ["index"]
     }),
     new HTMLWebpackPlugin({
-      filename: "room.html",
-      template: path.join(__dirname, "src", "room.html"),
-      chunks: ["room"],
+      filename: "hub.html",
+      template: path.join(__dirname, "src", "hub.html"),
+      chunks: ["hub"],
```

Comment:
Hmm. "hub" != "room"?

Reply:
yeah this page will be rendered 1-1 (now and forever I think) to the request to render a hub, regardless of if there is 1 virtual space to a hub or not